src/atldld/requests.py

```diff
@@ -0,0 +1,156 @@
+# The package atldld is a tool to download atlas data.
+#
+# Copyright (C) 2021 EPFL/Blue Brain Project
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""REST API requests to the AIBS servers.
+
+More info: https://help.brain-map.org/display/api/Allen+Brain+Atlas+API
+"""
+import logging
+from dataclasses import dataclass, replace
+from typing import Any, Dict, Optional, Sequence, Tuple
```

Comment:
Maybe we can add `mypy` in `setup.py`. I know it it in `tox.ini` but just to be consistent.

Reply:
Done in d7d2e25

Reply:
(Can be resolved)